Reconstruction/RecCalorimeter/tests/options/geant_fullsim_combinedCalo_singleparticles.py

```diff
@@ -0,0 +1,178 @@
+import os
+import numpy as np
+
+#loads array of random seeds from file                                                                                                                                                                          
+#seed_array = np.loadtxt('/afs/cern.ch/user/c/cneubuse/FCCSW/condor/seeds.txt',dtype='int',delimiter=',')
+
+#set these in the .sh script                                                                                                                                                                                                                
+energy=100000
```

Comment:
Maybe useful to use units here? e.g. `from GaudiKernel.SystemOfUnits import GeV, MeV`